spyder/widgets/about.py

```diff
@@ -36,33 +39,65 @@ def __init__(self, parent):
             revlink = " (<a href='https://github.com/spyder-ide/spyder/" \
-                      "commit/%s'>Commit: %s</a>)" % (rev, rev)
+                      "commit/%s'>Commit: <br> %s</a>)" % (rev, rev)
 
         # Get current font properties
         font = self.font()
         font_family = font.family()
-        font_size = font.pointSize()
-        if sys.platform == 'darwin':
-            font_size -= 2
-
-        self.label = QLabel((
+        buttons_padding = DialogStyle.ButtonsPadding
+        buttons_font_size = DialogStyle.ButtonsFontSize
+        font_size = DialogStyle.ContentFontSize
+        dialog_background_color = QStylePalette.COLOR_BACKGROUND_2
+        self.label_overview = QLabel((
             """
             <div style='font-family: "{font_family}";
-                        font-size: {font_size}pt;
+                        font-size: {font_size};
                         font-weight: normal;
                         '>
-            <p>
-            <b>Spyder {spyder_ver}</b> {revision}
             <br>
+            <p>
+            <b> Spyder IDE</b>
+            <br> <br>
             The Scientific Python Development Environment |
             <a href="{website_url}">Spyder-IDE.org</a>
             <br>
-            Copyright &copy; 2009-2020 Spyder Project Contributors and
-            <a href="{github_url}/blob/master/AUTHORS.txt">others</a>.
-            <br>
-            Distributed under the terms of the
-            <a href="{github_url}/blob/master/LICENSE.txt">MIT License</a>.
+            <p>
+            Python {python_ver} {bitness}-bit | Qt {qt_ver} |
+            {qt_api} {qt_api_ver} | {os_name} {os_ver}
             </p>
+            <br> <br>
+            <a href="{github_url}">GitHub </a>| <a href="{twitter_url}">
+            Twitter </a>|
+            <a href="{facebook_url}">Facebook </a>| <a href="{youtube_url}">
+            YouTube </a>|
+            <a href="{instagram_url}">Instagram </a>
+
+            </div>""").format(
+                website_url=__website_url__,
+                github_url=__project_url__,
+                twitter_url="https://twitter.com/Spyder_IDE",
+                facebook_url="https://www.facebook.com/SpyderIDE",
+                youtube_url="https://www.youtube.com/Spyder-IDE",
+                instagram_url="https://www.instagram.com/spyderide/",
+                python_ver=versions['python'],
+                bitness=versions['bitness'],
+                qt_ver=versions['qt'],
+                qt_api=versions['qt_api'],
+                qt_api_ver=versions['qt_api_ver'],
+                os_name=versions['system'],
+                os_ver=versions['release'],
+                font_family=font_family,
+                font_size=font_size,
+            )
+        )
+
+        self.label_community = QLabel((
+            """
+            <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
```

Comment:
```suggestion
                        font-size: {font_size};
```

`pt` is already included in `DialogStyle.ContentFontSize`